plugin/r/src/main/java/com/twosigma/beaker/r/utils/RServerEvaluator.java

```diff
@@ -153,6 +156,15 @@ public void evaluate(SimpleEvaluationObject seo, String code) {
     return myWorker.autocomplete(code,caretPosition);
   }
 
+  public void resetEnvironment() {
```

Comment:
should create a new RServerEvaluator (like calling getShell).
RESET_CODE is not guaranteed to reset all state.